website/docs/configuration/resources.html.md

```diff
@@ -46,21 +46,37 @@ arguments defined specifically for [the `aws_instance` resource type](/docs/prov
 
 Each resource is associated with a single _resource type_, which determines
 the kind of infrastructure object it manages and what arguments and other
-attributes are supported for each resource.
+attributes the resource supports.
 
-Each resource type in turn belongs to a [provider](/docs/configuration/providers.html),
-which is a plugin for Terraform that offers a collection of resource types that
-most often belong to a single cloud or on-premises infrastructure platform.
+Each resource type in turn belongs to a [provider](./providers.html),
+which is a plugin for Terraform that offers a collection of resource types. A
+provider usually provides resources to manage a single cloud or on-premises
+infrastructure platform.
 
-Most of the items within the body of a `resource` block are defined by and
-specific to the selected resource type, and these arguments can make full
-use of [expressions](/docs/configuration/expressions.html) and other dynamic
-Terraform language features.
+Most of the items within the body of a `resource` block are specific to the
+selected resource type. These arguments can make full use of
+[expressions](./expressions.html) and other dynamic Terraform
+language features.
 
-However, there are some "meta-arguments" that are defined by Terraform itself
-and apply across all resource types. These arguments often have additional
-restrictions on what language features can be used with them, and are described
-in more detail in the following sections.
+There are also some _meta-arguments_ that are defined by Terraform itself
+and apply across all resource types. (See [Meta-Arguments](#meta-arguments) below.)
+
+### Documentation for Resource Types
+
+[Terraform's provider documentation][providers] is the primary place to
+learn which resource types are available and which arguments to use for each
+resource type. Once you understand Terraform's basic syntax, the provider
+documentation will be where you spend the majority of your time on this website.
+
+The "[Providers][]" link at the top level of the navigation sidebar will take
```

Comment:
I'm not familiar with this link syntax... does this mean to use the link text itself (case-insensitively, I assume) to find the named link target further down the page? Good if so, just want to make sure!

Reply:
@apparentlymart Yeah! The original markdown spec calls this the ["the implicit link name shortcut"](https://daringfireball.net/projects/markdown/syntax#link) (cmd-f for the name, that page doesn't have very good linkable anchors), and the GFM spec calls it a ["collapsed reference link"](https://github.github.com/gfm/#collapsed-reference-link). In every implementation I've seen, they act case-insensitively, which is convenient.